flight/Modules/Actuator/actuator.c

```diff
@@ -61,6 +61,7 @@
 #define TASK_PRIORITY PIOS_THREAD_PRIO_HIGHEST
 #define FAILSAFE_TIMEOUT_MS 100
 #define MAX_MIX_ACTUATORS ACTUATORCOMMAND_CHANNEL_NUMELEM
+#define MULTIROTOR_MIXER_UPPER_BOUND 128
```

Comment:
(Not on the flight side-- on the GCS side.  Presumably appropriate saturation is in place so that we don't accidentally get the opposite mixer value than we'd like, but I'll verify that.)